REFERENCE.md

```diff
@@ -93,6 +94,40 @@ Data type: `Any`
 
 The data for when there is no cache yet
 
+### `extlib::cidr_to_network`
+
+Type: Ruby 4.x API
+
+Imported by Tim 'bastelfreak' Meusel into voxpupuli/extlib because Yelp/netstdlib got abandoned
+
+#### Examples
+
+##### calling the function
+
+```puppet
+extlib::cidr_to_broadcast('127.0.0.1/8')
```

Comment:
copy paste error? shouldn't this be `cidr_to_network`?

Reply:
thanks, fixed it